lib/cleaners/revisions_cleaner.rb

```diff
@@ -31,23 +51,17 @@ def self.repair_orphan_revisions
     end
   end
 
-  def self.find_orphan_revisions
-    article_ids = Article.all.pluck(:id)
-    orphan_revisions = Revision.where
-                       .not(article_id: article_ids)
-                       .order('date ASC')
-
-    Rails.logger.info "Found #{orphan_revisions.count} orphan revisions"
-    orphan_revisions
-  end
+  private
 
-  def self.before_earliest_revision(revisions)
-    date = revisions.first.date - 1.day
+  def before_earliest_revision(revisions)
```

Comment:
Doesn't depend on instance state (maybe move it to another class?) (UtilityFunction)